### ISSUE TITLE
perplexity = nan

### ISSUE THREAD

**Issue by EtienneAb3d** [ModernMT/MMT#382]
In a new training, without anything very different from the past, except a new language pair FR->ES, all was going well for few days, when suddenly the log is showing a calculation result "perplexity = nan" for each validation step. This activates the flag "stalled", and the learning rate is decreasing each time. With my configuration, if this is the only trouble, this should not be too damageable. But it's very strange...

**Comment by EtienneAb3d** [comment]
Hi,
I didn't know about the switching from OpenNMT to Tensor2Tensor/Tensorflow. When did this occur ?
As far as I see, the official latest 2.5 release is still with OpenNMT. No ?
How can I be informed of all changes in MMT ?
Best regards,
Etienne

**Comment by davidecaroselli** [comment]
Hi @EtienneAb3d 

there is a discussion, here on issue #377 , where you can find all the info about the switch and the new ModernMT with Tensor2Tensor Transformer. We are about to make the 3.0 release after having solved some issues and some deep testing.

To be informed of any discussions, you can "Star" the project (top right of the Github Page) and "Watch" the project selecting option "Watching". You'll be notified of issues, releases and discussions.

Best,
Davide
